akka-bench-jmh/src/main/scala/akka/actor/BenchmarkActors.scala

```diff
@@ -93,12 +95,13 @@ object BenchmarkActors {
     def props(next: Option[ActorRef]) = Props(new Pipe(next))
   }
 
+  @silent("deprecated")
```

Comment:
does this method need to use a deprecated api?

Reply:
Earlier deprecated was the pattern but on a retry with all the others in it does not which is mysterious.